podman/domain/containers.py

```diff
@@ -63,33 +70,64 @@ def attach(self, **kwargs) -> Union[str, Iterator[str]]:
              stream (bool): Return iterator of string(s) vs single string. Default: False
              logs (bool): Include previous container output. Default: False
         """
-        return self.client.api.attach(self.id, **kwargs)
+        raise NotImplementedError()
 
     def attach_socket(self, **kwargs):
         """TBD."""
-        raise NotImplementedError
+        raise NotImplementedError()
 
     def commit(self, repository: str = None, tag: str = None, **kwargs) -> Image:
         """Save container to given repository using given parameters.
 
         Args:
-            repository: Where to save Image.
-            tag: tag to push with Image.
+            repository: Where to save Image
+            tag: tag to push with Image
 
         Keyword Args:
-            message (str): Commit message to include with Image.
-            author (str): Name of commit author.
-            changes (str): Instructions to apply during commit.
-            conf (Dict[str, Any]): Configuration for the container.
+            author (str): Name of commit author
+            changes (List[str]): Instructions to apply during commit
+            comment (List[str]): instructions to apply while committing in Dockerfile format
+            conf (Dict[str, Any]): ignored
+            format (str): format of the image manifest and metadata
+            message (str): Commit message to include with Image
+            pause (bool): pause the container before committing it
```

Comment:
Nit for consideration.  Some of the words have the first letter uppercased above after the semi-colon, others are not.  I'd prefer all to be uppercased, but more so, consistent.